fhirResourcesToLoad/o2teddy_medicationstatement_azathio.json

```diff
@@ -0,0 +1,165 @@
+{
```

Comment:
Why use MedicationStatement not MedicationRequest?

Reply:
MedicationStatement is used by the patient to state he/she has been prescribed certain medication in the past as it is not necessary stored in the current EHR system as order/dispense request.

Reply:
I understand that. My question is why this information is obtained from Patient not from existing subscription? Is this the intention for the workflow? This MedicationStatement does reference to an existing MedicationRequest.

Reply:
Based on my understanding is that MedicationRequest is created by the provider when they want to prescribe certain medication for the patient and it is not necessary true that the Medication has been taken by the patient. While the MedicationStatement is more of the record for the medication the patient has taken over the course.  For this form, the latter one fits better and it is consistent with the master spreadsheet Carmela created.

Reply:
The difference is that MedicationStatement is statement from patient about medication usage. MedicationRequest is the prescription records for a physician. If the whole intention here is to obtain medication usage from patient, that is fine.

Reply:
Yes, the intention is to use the patient's input for Medications he/she has used.